Nickvision.Aura/Keyring/SystemCredentialManager.cs

```diff
@@ -0,0 +1,96 @@
+using Meziantou.Framework.Win32;
+using System;
+using System.Runtime.InteropServices;
+
+namespace Nickvision.Aura.Keyring;
+
+/// <summary>
+/// Object to access system credential manager
+/// </summary>
+/// <remarks>Uses Windows Credential Manager on Windows and LibSecret on Linux</remarks>
+public static partial class SystemCredentialManager
```

Comment:
This should be an `internal static partial class`

Reply:
This can be used on its own without keyring. Not sure we'll ever use it like this, but still, it's possible, if it's public.

Reply:
The thing is this was made for Keyring and like you said we will never use it someplace else so I think it's best to keep it `internal` until that time...if it every comes.

And if for any reason we need to use it besides Keyring it will probably be wrapped by some other class in Aura anyway so...